scripts/gen-dockerfile.sh

```diff
@@ -99,5 +131,3 @@ EXPOSE 20160 20180
 
 ENTRYPOINT ["/tikv-server"]
 EOT
-
-cat .gitignore > .dockerignore
```

Comment:
@overvenus PTAL, I think it's OK to let these diverge so we can optimize our `.dockerignore` later.